devicon.json

```
@@ -8644,6 +8644,24 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "titaniumsdk",
+        "altnames": [],
+        "tags": [
+            "app",
+            "mobile"
+        ],
```

Some more related tags.
```suggestion
        "tags": [
            "app",
            "mobile",
            "javascript",
            "cross-platform",
            "mobile",
            "sdk"
        ],
```

---

thats a great idea :+1: Added them